utils/build/docker/dotnet/parametric/ddtracer_version.Dockerfile

```
@@ -1,11 +1,10 @@
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0
+FROM mcr.microsoft.com/dotnet/sdk:8.0
 
 RUN apt-get update && apt-get install dos2unix
-
 WORKDIR /app
 
-COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries* /binaries/
+COPY install_ddtrace.sh query-versions.fsx binaries* /binaries/
```

The build context used for this Docker build is set to the base folder, so you'll need to include the relative directories like this:

```suggestion
COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries* /binaries/
```